### PR TITLE
Subpack changes

### PR DESCRIPTION
**Pull request by OpenSauce04**
### Summary:
- Made adjustments to the descriptions of every subpack to be more descriptive
- Removed the `custom` subpack due to being made completely redundant by how subpacks actually work (I will explain in a second comment)

### CODE REVIEW

**Comment by OpenSauce04** [comment]
As has been planned for a while, the preset system is going to rely on extra packs being loaded on top of OSBES's base files
I assume that you thought that these custom packs would need to be loaded with an identical filesystem to OSBES's, but with subpacks this is actually not the case:
MCdiagramThing
When Minecraft loads a subpack, it loads all of the files in the base pack, then loads the subpack's files as if it was a completely seperate pack **on top** of the original pack's files.
That means that if you had subpack `a` chosen, the files that are in `subpacks/a` (as far as the game is concerned) completely overwrite the files that were in the original pack

TLDR: This means that the existance of a `custom` option is unnecissary, as picking any other quality would just use that quality's base files instead of breaking the shader

**Comment by jebbyk** [comment]
no, "custom" subpack was made for those who dont want to place additional packages on top of the OSBES but just want to edit "custom" settings files by hands.

**Comment by jebbyk** [comment]
Preset is in different directory (/preset) and is not conflicting with "custom" subpack cuz quality settings are in /quality_settings next to the /preset.

**Comment by jebbyk** [comment]
I mean /quality_settings is not inside /preset directory

**Comment by OpenSauce04** [comment]
I still think it's unnecessary, especially as I've prepared a template and a guide on how to set up presets
You can see it if you'd like, it's privated on the OSBESdl GitHub profile atm
